kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/base/ThrowableUtil.java

```diff
@@ -31,9 +32,11 @@ private ThrowableUtil() {
      *
      * @param throwable
      *            the throwable to propagate
+     * @throws RuntimeException
+     *             translates the provided exception as a runtime exception
      */
     public static void propagate(final Throwable throwable) {
-        throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, throwable);
+        throw new RuntimeException(throwable);
```

Comment:
True. But you could simple inline that method. I see no real benefit in adding a method for that.